src/main/java/com/epam/grid/engine/provider/utils/CommandsUtils.java

```diff
@@ -28,15 +28,20 @@
 import java.util.List;
 
 import static com.epam.grid.engine.utils.TextConstants.NEW_LINE_DELIMITER;
+import static com.epam.grid.engine.utils.TextConstants.EMPTY_STRING;
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
-public final class SgeCommandsUtils {
+public final class CommandsUtils {
 
     private static final String MUST_BE_MANAGER = "must be manager for this operation";
 
     public static void throwExecutionDetails(final CommandResult result, final HttpStatus httpStatus) {
         final String message = result.getStdOut().isEmpty() ? mergeOutputLines(result.getStdErr()) :
-                mergeOutputLines(result.getStdOut()) + NEW_LINE_DELIMITER + mergeOutputLines(result.getStdErr());
+                mergeOutputLines(result.getStdOut())
+                        + (result.getStdErr().isEmpty()
+                        ?
```

Comment:
Let's cleanup formatting a bit:
```
    + (result.getStdErr().isEmpty()
    ? EMPTY_STRING
    : NEW_LINE_DELIMITER + mergeOutputLines(result.getStdErr())
```
Usually, for ternary operators we use one-line formatting (if it is possible) or the following:
```
... <condsition>
    ? <if true>
    : <or else>
```

Reply:
fixed